runtime-sdk/src/module.rs

```diff
@@ -256,6 +258,25 @@ impl BlockHandler for Tuple {
     }
 }
 
+/// Invariant handler.
+pub trait InvariantHandler {
+    /// Check invariants.
+    fn check_invariants<C: Context>(_ctx: &mut C) -> Result<(), CoreError> {
```

Comment:
```suggestion
    fn check_invariants<C: Context>(_ctx: &mut C) -> Result<(), modules::core::Error> {
```
(To be consistent with other places.)